test/integration/supervisor_upstream_test.go

```diff
@@ -84,6 +85,7 @@ oidc: issuer did not match the issuer returned by provider, expected "` + env.Su
 				Reason:  "Success",
 				Message: "additionalAuthorizeParameters parameter names are allowed",
 			},
+			expectedTLSConfigValidCondition(env.SupervisorUpstreamOIDC.CABundle != ""),
```

Comment:
Same comment as in supervisor_login_test... I wonder if this could be refactored somehow so it does not need to make assumptions about the internals of the `testlib.CreateTestOIDCIdentityProvider` helper?

Reply:
Oops, ignore this comment. Deleting it.